typo3-fractor/rules/TYPO3v10/Yaml/TranslationFileFractor.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Typo3Fractor\TYPO3v10\Yaml;
+
+use a9f\FractorYaml\Contract\YamlFractorRule;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final class TranslationFileFractor implements YamlFractorRule
+{
+    /**
+     * @var string
+     */
+    private const TRANSLATION_FILE_KEY = 'translationFile';
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Use key translationFiles instead of translationFile', [
```

Comment:
IMO this should have some reference to EXT:form in it, e.g. "Rewrite EXT:form configuration files to use key "translationFiles" instead of "translationFile" in renderingOptions."